### PR TITLE
Fix `pickBy` and `omitBy` method definitions from `lodash` so that they correctly accept objects with `string` and `number` typed ids.

### PR DESCRIPTION
**Pull request by tyger**
In response to the next issue #2623

### CODE REVIEW

**Comment by davidgomes** [comment]
Great find. I can confirm that my PR #2613 was indeed correct and this is the right way to do it.

File: cli/src/commands/runTests.js [hunk 1]
```diff
@@ -679,7 +679,7 @@ export function setup(yargs: Yargs) {
 }
 
 export async function run(argv: Args): Promise<number> {
-  if (!(await fs.exists(BIN_DIR))) {
+  if (!await fs.exists(BIN_DIR)) {
```
Comment (tyger) [inline cli/src/commands/runTests.js]: This change because of `eslint` complaint about extra parenthesis.

**Comment by tyger** [comment]
@davidgomes  Thank you.

**Comment by villesau** [comment]
Thanks for the contributions @tyger and @davidgomes !

**Pull request merged**